website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+### Setup Recommendations
```

Comment:
I don't think this information belongs on the configs doc. The configs doc should be purely model configs. I'd remove this section altogether and just leave it in the adapter repo's README.

Reply:
@swanderz ?

Reply:
In dbt docs, this page will be listed as a sub-page of "Adapter-specific configs". `quote_columns` isn't adapter specific.

Reply:
I personally think this would be good to leave in the adapter-specific config because you can give a warning as to why the warning is happening and how to fix it (especially because we don't have adapter specific troubleshooting pages)

Reply:
Okay, that's fine with me. Let's just leave the quote_columns thing in there then.